app/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/disposaldetails/DisposalDetailsController.scala

```diff
@@ -290,7 +291,15 @@ class DisposalDetailsController @Inject() (
                     i.copy(
                       disposalDetailsAnswers = Some(newAnswers),
                       acquisitionDetailsAnswers =
-                        i.acquisitionDetailsAnswers.map(_.unsetAllButAcquisitionMethod(i.triageAnswers)),
+                        if (fillingOutReturn.isFurtherReturn.contains(true))
```

Comment:
`shouldUseRebase` should be cleared as well - it looks like you need to set this section to `None` for a further return

Reply:
`gainorLossAfterReliefs` should be unset here and in the other questions in this section as well